iOSClient/Viewer/NCViewerMedia/NCPlayer/NCPlayerToolBar.swift

```diff
@@ -359,7 +385,76 @@ class NCPlayerToolBar: UIView {
 
         timerAutoHide?.invalidate()
     }
-
+    
+    // MARK: - Message
+    
+    func showMessage(_ title: String, description: String?, backgroundColor: UIColor = NCBrandColor.shared.brand, isHiddenPregress: Bool = true, hiddenAfterSeconds: Double = 0) {
```

Comment:
`isHiddenPregress` should be `isProgressHidden`

Reply:
ok, next

Reply:
this func is in progress, I don't like it yet.